pkgs/by-name/im/imgbrd-grabber/package.nix

```diff
@@ -35,43 +30,46 @@ stdenv.mkDerivation rec {
     typescript
   ];
 
-  nativeBuildInputs = [
-    makeWrapper
-    qtmultimedia
-    qtbase
-    qtdeclarative
-    qttools
-    qtnetworkauth
-    nodejs
-    cmake
-    wrapQtAppsHook
-  ];
+  nativeBuildInputs =
+    with qt6;
+    [
+      qtbase
+      qtdeclarative
+      qttools
+      qtnetworkauth
+      qtmultimedia
+      wrapQtAppsHook
+    ]
+    ++ [
+      makeWrapper
+      nodejs
+      cmake
+    ];
 
   extraOutputsToLink = [ "doc" ];
 
-  postPatch = ''
+  preBuild = ''
+    export HOME=$TMPDIR
+
     # the package.sh script provides some install helpers
     # using this might make it easier to maintain/less likely for the
     # install phase to fail across version bumps
-    patchShebangs ./scripts/package.sh
+    patchShebangs ../scripts/package.sh
+  '';
 
+  patchPhase = ''
```

Comment:
Not only will not postPatch run now but also patches will be broken. This should all be moved to postPatch